### PR TITLE
Add status condition

### PR DESCRIPTION
**Pull request by tmihalac**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,6 +38,7 @@
         <module>mercury-api</module>
         <module>mercury-framework</module>
         <module>mercury-operator</module>
+        <module>integrations</module>
```
Comment (ruromero) [inline pom.xml]: this project doesn't require mvn build anymore

File: pom.xml [hunk 2]
```diff
@@ -17,12 +17,14 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.platform.version>2.6.3.Final</quarkus.platform.version>
+        <quarkus.platform.version>2.7.0.Final</quarkus.platform.version>
+        <quarkus.camel.version>2.7.0.CR1</quarkus.camel.version>
+        <quarkus.camel.version>2.7.0.CR1</quarkus.camel.version>
```
Comment (ruromero) [inline pom.xml]: Duplicated line

File: .github/workflows/codeql-analysis.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
```
Comment (ruromero) [inline .github/workflows/codeql-analysis.yaml:1]: I don't think this is part of this PR

**Review by ruromero — CHANGES_REQUESTED** [review]
First iteration

File: .github/workflows/codeql-analysis.yaml (hunk 3 above)
Comment (tmihalac) [reply .github/workflows/codeql-analysis.yaml]: removed

**Pull request closed without merging**